roles/aap_setup_download/README.md

```diff
@@ -0,0 +1,55 @@
+aap\_setup\_download
+==================
+
+A role to download the latest z-version of the AAP setup tarball for a given minor version (e.g. 2.1 at time of writing).
+
+Shamelessly adapted from [Red Hat Ansible Automation Platform 2: Automating the Installer Download and publishing as a Content View in Satellite](https://www.redhat.com/en/blog/automating-installation-ansible-automation-platform-ansible-and-satellite).
+
+Requirements
+------------
+
+You will need a Red Hat Ansible Automation Platform (AAP, hence the name) subscription.
+Once this is a given, you will be able to create yourself an offline token
+
+Role Variables
+--------------
+
+The following input variables are required:
+
+* `aap_setup_down_offline_token` contains your offline token as described in the requirements.
```

Comment:
we should point the user in the right place to get the token, rather then going to the blog post. 
https://access.redhat.com/management/api is the link to follow.